lib/src/sturdy_http_event_listener.freezed.dart

```diff
@@ -69,7 +69,9 @@ mixin _$SturdyHttpEvent {
   }) =>
       throw _privateConstructorUsedError;
 
-  @JsonKey(ignore: true)
+  /// Create a copy of SturdyHttpEvent
+  /// with the given fields replaced by the non-null parameter values.
+  @JsonKey(includeFromJson: false, includeToJson: false)
```

Comment:
just curious why this freezed file changed so much when the only change in the base file was removing an import 🤔

Reply:
Likely because when I cloned and `pub get`'d, a different (higher) version of `freezed` was pulled than the one that generated the original file, and new code is generated with the newer version.

Because `pana` failed (version decreased) I'm looking at upgrading some packages anyway so that may help make that more intuitive.

Reply:
OK @ClaireDavis, should be good to go now!